docs/docs/tascomi-ingestion.md

```diff
@@ -0,0 +1,73 @@
+
+This process writes into the refined zone, in the 'snapshot' bucket. The data is partitioned by snapshot_date.
+
+# Full workflow and scheduling
+The full workflow is defined in the [glue-tascomi-data terraform script](https://github.com/LBHackney-IT/Data-Platform/blob/main/terraform/24-aws-glue-tascomi-data.tf). 
+It defines a list of tables that needs updating everyday, and a list of static tables that are only updated weekly (these are the static tables like application types). The schedule is as follows:
+- 3am: as many jobs as tables to update are triggered. Each job queries one API endpoint for latest updated records. That's 25 jobs on Sundays (including static tables), about half of that on other days.
+- 4am: a crawler crawls the API results bucket
+  - the previous crawler triggers the parsing job and the crawling of its results
+  - the previous crawler triggers the recasting job and the crawling of its results
+  - the previous crawler triggers the daily snapshot creation job and the crawling of its results
+- 5am: the API results bucket gets crawled again - this is in case the ingestion had not finished at 4am when the first crawling happened, and some tables were missed.
+  - the previous crawler triggers the parsing job and the crawling of its results. Most of the time, there won't be new data process so the job will finish early.
+  - the previous crawler triggers the recasting job and the crawling of its results. Most of the time, there won't be new data process so the job will finish early.
+  - the previous crawler triggers the daily snapshot creation job and the crawling of its results. Most of the time, there won't be new data process so the job will finish early.
+
+# Structure of the S3 buckets and Glue tables
+The data created along the process (initial full load, increments and snapshots) is stored in S3 in the raw and refined zones, with one folder per table.
+
+The ready-for-use data is in the Refined zone Planning bucket, in the snapshot area. The corresponding tables in the Glue catalog are simply called applications, appeals, etc. To get the latest data, the query must refer to the snapshot_date latest partition, for example 
+```
+select count(*) from "dataplatform-stg-tascomi-refined-zone"."applications" where snapshot_date = (select max(snapshot_date) from "dataplatform-stg-tascomi-refined-zone"."applications")
```

Comment:
This query would be to get the number of records for the latest date rather than the latest data. Can just be `Select * FROM ..`

Reply:
done